docs/configuration/arguments.md

```diff
@@ -150,11 +150,13 @@ The ingester query API was improved over time, but defaults to the old behaviour
 
 ### Ring/HA Tracker Store
 
-The KVStore client is used by both the Ring and HA Tracker.
+The KVStore client is used by both the Ring and HA Tracker (HA Tracker doesn't support memberlist as KV store).
 - `{ring,distributor.ha-tracker}.prefix`
    The prefix for the keys in the store. Should end with a /. For example with a prefix of foo/, the key bar would be stored under foo/bar.
 - `{ring,distributor.ha-tracker}.store`
-   Backend storage to use for the ring (consul, etcd, inmemory, memberlist, multi).
+   Backend storage to use for the HA Tracker (consul, etcd, inmemory).
+- `{ring,distributor.ring}.store`
+   Backend storage to use for the Ring (consul, etcd, inmemory, memberlist, multi)).
```

Comment:
Looks duplicated.

Reply:
I'm not sure, for me the first one is for the HA Tracker and the second one for the distributor ring. For instance, we can have the distributor in memberlist with its HA Tracker using ETCD. Tell me if it's a bad practice to do so?

Reply:
Doh! I totally missed it, sorry.